### PR TITLE
CSS Namespacing

### PR DESCRIPTION
**Pull request by kkozee**
@KyleAMathews

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,6 +23,6 @@ build/Release
 # Deployed apps should consider commenting this line out:
 # see https://npmjs.org/doc/faq.html#Should-I-check-my-node_modules-folder-into-git
 node_modules
-dist
```
Comment (KyleAMathews) [inline .gitignore]: Why was this removed?

Comment (KyleAMathews) [inline .gitignore]: The compiled code shouldn't be committed to Github. I compile when creating a new release and push that to NPM. I think compiled code in Github is a bit icky as it creates weird diffs plus is hard to remember to keep updated plus isn't that useful anyways since you only need the compiled version when installing.

Comment (kkozee) [inline .gitignore]: @KyleAMathews I agree; that was leftover from the original PR. I didn't alter it on the off chance that I was missing something but since you've confirmed the compiled code shouldn't be there, I'll update the PR.

**Comment by KyleAMathews** [comment]
Published a 2.0! Thanks @kkozee and @carlos-stored!
